internal/pkg/tui/installer/installer.go

```diff
@@ -205,7 +205,7 @@ func (installer *Installer) configure() error {
 
 	capture := installer.app.GetInputCapture()
 	installer.app.SetInputCapture(func(e *tcell.EventKey) *tcell.EventKey {
-		//nolint:exhaustive
+		//nolint:exhaustive,nolintlint
```

Comment:
I feel we need to get to the bottom of this issue, it shouldn't be this way

Reply:
Yeah, that's _really_ nasty

Reply:
Reverted those thing from the push.  It fails locally, but I'm curious to see if it works in CI.

Reply:
Clean on CI. Yuck